src/widgets/hierarchical-menu/__tests__/hierarchical-menu-test.ts

```diff
@@ -133,18 +133,33 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hierarchica
 
       expect(results.getFacetValues).toHaveBeenCalledTimes(1);
       expect(results.getFacetValues).toHaveBeenCalledWith('hello', {
+        facetOrdering: true,
         sortBy: ['name:asc'],
       });
     });
 
     it('has a sortBy option', () => {
-      widget = hierarchicalMenu({ ...options, sortBy: ['name:asc'] });
+      widget = hierarchicalMenu({ ...options, sortBy: ['name:desc'] });
```

Comment:
Is there a reason for needing this test to change to `name:desc`.

Reply:
so that it's not the same as the default value

Reply:
oh that makes sense.